tenable/io/v3/vm/api.py

```diff
@@ -37,6 +39,14 @@ class VulnerabilityManagement(APIEndpoint):  # noqa: PLR0904
     i.e assets, agents, scanners etc.
     '''
 
+    @property
```

Comment:
Please follow the ASCII order. So, Add agent_groups after agent_config, similar to the order in imports.

Reply:
update the changes as per comment.